python/ql/test/query-tests/Functions/return_values/ReturnConsistentTupleSizes.expected

```diff
@@ -1,2 +1 @@
 | functions_test.py:306:1:306:39 | Function returning_different_tuple_sizes | returning_different_tuple_sizes returns $@ and $@. | functions_test.py:308:16:308:18 | Tuple | tuple of size 2 | functions_test.py:310:16:310:20 | Tuple | tuple of size 3 |
```

Comment:
I might like to put some comments in the tests explaining the expected results.
(in particular that this case no longer gives an alert)

Reply:
Good idea. I have done so in [6674288].(https://github.com/github/codeql/pull/19136/commits/6674288fd2bdb9425e3e64ae4eb83711b096b6c8)